lineman/app/components/thread_page/activity_card/activity_card.haml

```diff
@@ -6,7 +6,7 @@
   %loading.activity-card__loading.page-loading{ng-show: 'loadEventsBackwardsExecuting'}
   %ul.activity-card__activity-list
     %li.activity-card__activity-list-item{ng_repeat: 'event in discussion.events() track by event.id', in-view: '($inview&&threadItemVisible(event)) || (!$inview&&threadItemHidden(event))', in-view-options: '{debounce: 100}'}
-      .activity-card__new-activity{ng-if: 'event.sequenceId == newActivitySequenceId', translate: 'activity_card.new_activity'}
       .activity-card__last-item{ng-if: '$last'}
       .activity-card-content{id: 'sequence-{{event.sequenceId}}', ng-include: "'generated/components/thread_page/activity_card/'+ event.kind + '.html'"}
+      .activity-card__new-activity{ng-if: 'event.sequenceId == lastReadSequenceId', translate: 'activity_card.new_activity'}
```

Comment:
If the new activity indicator is rendered after the record, then we can pop straight to it without worrying about whether the sequenceId after it is rendered or not.